tasks/places.rb

```diff
@@ -0,0 +1,14 @@
+require 'yaml'
+require './models.rb'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.